src/UWP/ArcGISRuntime.UWP.Viewer/Samples/MapView/DisplayDrawingStatus/DisplayDrawingStatus.xaml

```diff
@@ -9,8 +9,18 @@
     d:DesignHeight="300"
     d:DesignWidth="400">
     <Grid>
-        <esriUI:MapView x:Name="MyMapView"/>
-        <ProgressRing x:Name="activityIndicator" VerticalAlignment="Bottom" HorizontalAlignment="Center"
-                      Margin="0,0,0,100" IsActive="True"/>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border BorderBrush="Black" BorderThickness="1" Background="White"
+                VerticalAlignment="Top" HorizontalAlignment="Right"
+                Margin="30" Padding="20" Width="300">
+            <StackPanel>
+                <TextBlock x:Name="StatusDisplay"
+                           TextAlignment="Center" FontWeight="SemiBold" />
+                <ProgressRing x:Name="ActivityIndicator"
```

Comment:
I feel like this could use some padding or rearrangement. Not a very appealing layout at the moment.

![2018-06-12_11-19-37_applicationframehost](https://user-images.githubusercontent.com/587809/41309004-515b9a1a-6e32-11e8-9e33-5d1e5b92d4dd.png)